src/models/user.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+const { Model } = require("sequelize");
```

Comment:
try to use ES6 instead of ES5

Reply:
This is an auto-generated code from sequelize. If you know how to convert this line in ES6 please assist.

Reply:
Resolved